### PR TITLE
Contributions from Super Senior :D 

### PR DESCRIPTION
**Pull request by kritishn143**
Refactored code for ATM program, added input buffer clearance for invalid PIN entry, displayed 'Successfully inserted ATM Card' on startup, and improved PIN validation during PIN change.

From @aghori-io Super Senior

### CODE REVIEW

**Comment by Anmol2059** [comment]
**@kritishn143 pls fix,** 
The code provided contains a mix of functional code and areas that need improvement. Here's a review with some comments:

_Issues and Suggestions:_

- [ ] 1. GOTO Statements:
The use of goto statements for control flow is generally discouraged as it can make the code hard to follow and maintain. It's better to use loops or functions to manage control flow.

- [ ] 2. Lack of Modularity:
The initial version of the code lacks modularity and readability. Breaking it into functions like withdraw, displayBalance, changePIN, etc., would improve its structure.

- [ ] 3. Variable Naming:
Variable names like inpPin, wdrawAmt, re are not very descriptive. More meaningful names like inputPIN, withdrawalAmount, repeatTransaction could enhance readability.

**@aghori-io** try to incorporate these and only pull and merge this request xD, 
_Review from : Super Ultra Pro Max Senior_

**Comment by aghori-io** [comment]
yes

**Review by aghori-io — APPROVED** [review]
Acknowledged & appreciated. Thanks!
